### DISCUSSION TITLE
Raspberry PI

### DISCUSSION THREAD

**Discussion by leesoyavh** [thirtythreeforty/neolink#175] [Q&A]
hi, is it possible to get this working on raspberry ? i just bought one today to run homebridge...

**Comment by leesoyavh** [comment]
hi, is it possible to get this working on raspberry ? i just bought one today to run homebridge...

**Comment by QuantumEntangledAndy** [comment]
Yes it is [here](https://nightly.link/thirtythreeforty/neolink/workflows/build/master/release-armhf-buster.zip)

You should follow the setup in `docs/unix_setup.md`, and if you search home assistant in the search bar at the top of this github page you will find how others have setup ha.
